finality-aleph/src/sync/forest/mod.rs

```diff
@@ -56,6 +56,20 @@ pub enum Interest<I: PeerId, J: Justification> {
     },
 }
 
+/// What kind of extension we should request and from whom.
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub enum ExtensionRequest<I: PeerId, J: Justification> {
+    /// We are not interested in requesting anything at this point.
+    None,
```

Comment:
I dont like `None` as its confusing to me with None from std option. We usually use Noop in finality-aleph in such cases.

Reply:
Right, what have I been thinking, changed.